src/platform/activity/activity.entity.ts

```diff
@@ -12,27 +16,27 @@ export class Activity extends BaseAlkemioEntity implements IActivity {
   @Generated('increment')
   rowId!: number;
 
-  @Column('char', { length: 36, nullable: false })
+  @Column('char', { length: UUID_LENGTH, nullable: false })
   triggeredBy!: string;
 
-  @Column('char', { length: 36, nullable: false })
+  @Column('char', { length: UUID_LENGTH, nullable: false })
   resourceID!: string;
 
-  @Column('char', { length: 36, nullable: true, default: '' })
-  parentID!: string;
+  @Column('char', { length: UUID_LENGTH, nullable: true })
+  parentID?: string;
 
-  @Column('char', { length: 36, nullable: false })
+  @Column('char', { length: UUID_LENGTH, nullable: false })
   collaborationID!: string;
 
-  @Column('char', { length: 44, nullable: true })
+  @Column('char', { length: MESSAGEID_LENGTH, nullable: true })
   messageID!: string;
 
-  @Column('boolean', { default: true })
-  visibility = true;
+  @Column('boolean', { nullable: false })
+  visibility!: boolean;
 
   @Column({
-    // TODO: It's 255, migrate it to something else
-    length: SMALL_TEXT_LENGTH,
+    length: 255, // already 255 in db
```

Comment:
Length via constant? Medium text length? This is a field we use in the code, not user provided data.